index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FCC: Tribute Page</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Tribute Page</h1>
+      <p>Человек, который изменил мир.</p>
+      <div id="img-div">
+        <img id="image" src="stive.jpg" alt="Steve Jobs" />
+        <figure id="img-caption">
+          Я не доверяю компьютеру, который не могу поднять (Стив Джобс).
+        </figure>
+      </div>
+      <section id="tribute-info">
+        <h3 id="heading">Краткая биография Стива Джобса:</h3>
+        <ul>
+          <li>
+            <strong>1955</strong> - Стивен Пол Джобс родился в городке Маунтин

```

Comment:
There is a better solution for bold font here. Imagine that one day I ask you to make all dates red and not-bold... Do you want to change it everywhere in your code manually?

Reply:
Corrected!